js/components/portal-dashboard/custom-select.tsx

```diff
@@ -1,12 +1,14 @@
 import React from "react";
+import ArrowIcon from "../../../img/svg-icons/arrow-icon.svg";
+import CheckIcon from "../../../img/svg-icons/check-icon.svg";
 
 import css from "../../../css/portal-dashboard/custom-select.less";
 
 interface IProps {
   items: SelectItem[];
   onSelectItem: (value: string) => void;
   trackEvent: (category: string, action: string, label: string) => void;
-  iconId: string;
+  HeaderIcon: any; // icon as react component
```

Comment:
You should be able to type this as:
```suggestion
  HeaderIcon: ReactNode; // icon as react component
```
if you change the import above to:
```
import React, { ReactNode } from "react";
```

Reply:
Nice!  I fiddled with this for a bit last night, but didn't see the solution.  This also let me fix the types in a couple of my utility modules.